api/dbadapters/bigquery.ts

```diff
@@ -190,6 +194,24 @@ export class BigQueryDbAdapter implements IDbAdapter {
 
   public async close() {}
 
+  public async persistedStateMetadata(
+    projectConfig: dataform.IProjectConfig
+  ): Promise<dataform.IPersistedTableMetadata[]> {
+    try {
+      const { rows } = await this.runQuery(
+        `SELECT * FROM \`${projectConfig.defaultDatabase}.${projectConfig.defaultSchema}.${STATE_PERSIST_TABLE_NAME}\``,
```

Comment:
Can we re-use the resolveTarget method from the adpater that already does this?

Reply:
done